common/oauthTokenManager.go

```diff
@@ -396,11 +412,7 @@ func (uotm *UserOAuthTokenManager) getTokenInfoFromEnvVar(ctx context.Context) (
 	}
 
 	if tokenInfo.TokenRefreshSource != TokenRefreshSourceTokenStore {
-		refreshedToken, err := tokenInfo.Refresh(ctx)
-		if err != nil {
-			return nil, fmt.Errorf("get token from environment variable failed to ensure token fresh, %v", err)
-		}
-		tokenInfo.Token = *refreshedToken
+		panic("Invalid Token Refresh Source")
```

Comment:
Why are we panicking here? Refreshing should mean re checking the environment variable.

Reply:
I need some context here - lets discuss offline on this.